app/services/homePagePropsService.ts

```diff
@@ -1,4 +1,4 @@
-import { HomePageProps } from "../pages/index";
+import { HomePageProps } from "../../pages/index";
```

Comment:
This now feels really wrong 😅 does TS have some sort of resolution that allows circular imports?

I think what we should do is make a type HomePageData, in types - definitely a next PR jobby